src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -242,7 +242,7 @@ public HttpResponse doSubmitContainerStatus(
         checkPermission(DockerTraceabilityPlugin.SUBMIT);
         final ObjectMapper mapper = new ObjectMapper();
         final InspectContainerResponse[] inspectContainerResponses = mapper.readValue(inspectData, InspectContainerResponse[].class);
-        final long eventTime = time != 0 ? time : new Date().getTime();
+        final long eventTime = time != 0 ? time : new Date().getTime()/1000;
```

Comment:
`System.currentTimeMillis()` would avoid the temporayObject creation.
Could also Use `java.util.concurrent.TimeUnit`s conversion to make it more explicit.